go.mod

```diff
@@ -288,6 +288,8 @@ require (
 	k8s.io/klog/v2 v2.90.1 // @grafana/grafana-app-platform-squad
 )
 
+require github.com/blang/semver/v4 v4.0.0
```

Comment:
I'm not sure where this dependency is coming from - I recreated this PR locally and it's not showing up. Do you have any ideas, or are you able to remove it?